SimPEG/electromagnetics/analytics/NSEM.py

```diff
@@ -65,7 +65,7 @@ def _Propagate(f, thickness, sig, chg, taux, c, mu_r, eps_r, n):
         UD = UD / ((np.abs(UD[0, :] + UD[1, :])).max())
 
     for j in range(0, n + 1):
-        EH[:, j] = np.matrix([[1.0, 1,], [-1.0 / Z[j], 1.0 / Z[j]],]) * UD[:, j]
+        EH[:, j] = np.array([[1.0, 1,], [-1.0 / Z[j], 1.0 / Z[j]],]) * UD[:, j]
```

Comment:
the "*" operator should be changed to an "@" operator, on this line, and on line 64 above. What's going to throw most of the errors here is that the multiplication for a `numpy.matrix` was actual matrix multiplication (which is completely different than the `ndarray` class, clearly confusing and why they are getting rid of the matrix class.)